docs/tutorials/the_command_line.md

```diff
@@ -1,25 +1,37 @@
 # The Command Line
+
 * Using the gem
 
-        $ beaker --log-level debug --hosts sample.cfg --tests test.rb
+```console
+    $ beaker --log-level debug --hosts sample.cfg --tests test.rb
+```
 
 * Using latest git
 
-        $ bundle exec beaker --log-level debug --hosts sample.cfg --tests test.rb
+```console
+    $ bundle exec beaker --log-level debug --hosts sample.cfg --tests test.rb
+```
 
 ## Useful options
+
 * `-h, --hosts FILE `, the hosts that you are going to be testing with
 * `--log-level debug`, for providing verbose logging and full stacktraces on failure
 * `--[no-]provision`, indicates if beaker should provision new boxes upon test execution. If `no` is selected then beaker will attempt to connect to the hosts as defined in `--hosts FILE` without first creating/running them through their hypervisors
 * `--preserve-hosts [MODE]`, indicates what should be done with the testing boxes after tests are complete.  If `always` is selected then the boxes will be preserved and left running post-testing.  If `onfail` is selected then the boxes will be preserved only if tests fail, otherwise they will be shut down and destroyed.  If `never` is selected then the boxes will be shut down and destroyed no matter the testing results.
 * `--parse-only`, read and parse all command line options, environment options and file options; report the parsed options and exit.
 
 ## The Rest
+
 See all options with
+
 * Using the gem
 
+```console
         $ beaker --help
+```
 
 * Using latest git
 
+```console
```

Comment:
💯 didn't know about console before this, cool.

Reply:
Yeah, always good to reference [GitHub's supported syntax highlighting languages](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) when writing Markdown, lots of supported languages and aliases. Console is the shortest alias for the bash/shell session mode. There's also `puppet`, which currently maps to plaintext but should get highlighting soon [per this just-merged pull to the Ace highlighter that GitHub uses](https://github.com/ajaxorg/ace/pull/3666).